### DISCUSSION TITLE
Generalized gamma parameters for covariates

### DISCUSSION THREAD

**Discussion by shantanuneema** [CamDavidsonPilon/lifelines#1401] [General]
@CamDavidsonPilon, I figured the problem I am working on falls in generalized gamma fitter. I do have several covariates and no matter what I always see this warning (even if I run without any covariates):

python3.8/site-packages/autograd_gamma/__init__.py:118: RuntimeWarning: invalid value encountered in true_divide

python3.8/site-packages/autograd_gamma/__init__.py:115: RuntimeWarning: invalid value encountered in subtract
  delta = temp - x

python3.8/site-packages/autograd_gamma/__init__.py:180: RuntimeWarning: invalid value encountered in subtract
  lambda g: g * np.exp(-x + np.log(x) * (a - 1) - gammaln(a) - gammaincln(a, x)),

I followed the example on page 94 with rossi dataset given in: https://buildmedia.readthedocs.org/media/pdf/lifelines/latest/lifelines.pdf

Does that mean any instability in the model or the warnings could be ignored?

**Comment by CamDavidsonPilon** [comment]
Hi @shantanuneema, 

The warnings can be ignored if the results look sensible. Do they look sensible?

**Reply by shantanuneema** [reply]
Yes, I do not see any issues with results presented in the example. Thanks for responding.

**Reply by shantanuneema** [reply]
@CamDavidsonPilon, same example in documentation (pg 94), the model `ggf.concondance_index_` shows this error:

`AttributeError: 'GeneralizedGammaRegressionFitter' object has no attribute '_predicted_median'`

**Comment by shantanuneema** [comment]
@CamDavidsonPilon, Thank you for adding the issue. One more thing I noticed was whenever I play with randomness in the dataset with train/test split, there are times when model does not converge. I think it relates to the earlier issue I raised about the warning. I played with publicly available dataset colon.csv from https://dmkd.cs.vt.edu/projects/survival/data/. I am unsure what is wrong, I will investigate on my own and update here.
